frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -1134,6 +1134,89 @@ describe("scenarios > question > filter", () => {
       cy.button("Visualize");
     });
   });
+
+  ["true", "false"].forEach(condition => {
+    const regexCondition = new RegExp(`${condition}`, "i");
+    // We must use and return strings instead of boolean and numbers
+    const integerAssociatedWithCondition = condition === "true" ? "0" : "1";
+
+    describe.skip(`should be able to filter on the boolean column ${condition.toUpperCase()} (metabase#16386)`, () => {
+      beforeEach(() => {
+        cy.createNativeQuestion({
+          name: "16386",
+          native: {
+            query:
+              'select 0::integer as "integer", true::boolean AS "boolean" union all \nselect 1::integer as "integer", false::boolean AS "boolean" union all \nselect null as "integer", true::boolean AS "boolean" union all \nselect -1::integer as "integer", null AS "boolean"',
+          },
+          visualization_settings: {
+            "table.pivot_column": "boolean",
+            "table.cell_column": "integer",
+          },
+        }).then(({ body: { id: QUESTION_ID } }) => {
+          cy.visit(`/question/${QUESTION_ID}`);
+          cy.findByText("Explore results").click();
+        });
+      });
+
+      it("from the column popover (metabase#16386-1)", () => {
+        cy.get(".cellData")
+          .contains("boolean")
+          .click();
+
+        popover()
+          .findByText("Filter by this column")
+          .click();
+
+        popover().within(() => {
+          // Not sure exactly what this popover will look like when this issue is fixed.
+          // In one of the previous versions it said "Update filter" instead of "Add filter".
+          // If that's the case after the fix, this part of the test will need to be updated.
```

Comment:
Quite sure the Column-type still says "Update filter", while the Sidebar says "Add filter" and Notebook says the correct one depending on new/existing filter.
Probably because Column uses the same logic as the breadcrumb filters, so it always expects them to be existing.